hudi-spark-datasource/hudi-spark2/pom.xml

```diff
@@ -203,7 +203,19 @@
       <optional>true</optional>
     </dependency>
 
+    <!-- Extra Logging Bridges -->
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>jul-to-slf4j</artifactId>
```

Comment:
so this is only needed for spark2?

Reply:
I think I can remove this now that we've added it to hudi-common.